packages/clerk-js/src/core/clerk.ts

```diff
@@ -129,8 +130,8 @@ const defaultOptions: ClerkOptions = {
   isSatellite: false,
   signInUrl: undefined,
   signUpUrl: undefined,
-  afterSignInUrl: undefined,
-  afterSignUpUrl: undefined,
+  afterSignInUrl: '/',
+  afterSignUpUrl: '/',
```

Comment:
I need to take another look, but my instinct tells me that this is not the direction we want to go as this will cause these to have actual values so, by default, the URLs will include the query params.

What we want to do instead, is to change the behavior of Clerk IF these are undefined

Reply:
Made the changes to only fallback to `/` and the values will not be included to the query params.